riptide-core/src/main/java/org/zalando/riptide/RestBuilder.java

```diff
@@ -84,6 +90,14 @@ public static RestConfigurer simpleRequestFactory(final ExecutorService executor
         };
     }
 
+    @Override
+    public RestBuilder clone() {
+        return new RestBuilder()
+                .requestFactory(requestFactory)
+                .converters(converters)
+                .baseUrl(baseUrl);
+    }
+
     public Rest build() {
         return new Rest(requestFactory, converters(), baseUrl);
```

Comment:
If we make a copy of the list here the builder can be re-used without cloning it.